README_wasm_of_ocaml.md

```diff
@@ -4,7 +4,7 @@ Wasm_of_ocaml is a fork of Js_of_ocaml which compiles OCaml bytecode to WebAssem
 
 ## Supported engines
 
-The generated code works with Chrome 11.9, Node.js 22 and Firefox 122 (or more recent versions of these applications).
+The generated code works with Chrome 119, Node.js 22 and Firefox 122 (or more recent versions of these applications).
```

Comment:
@vouillon, is this version correct ? we install node 23 in .github/workflows/build-wasm_of_ocaml.yml.

Reply:
We need Chrome 126 and node 23 only for the [JSPI](https://v8.dev/blog/jspi)-based implementation of effects. JSPI is still experimental and not enabled by default.
These older versions should work otherwise.